packages/notarization/storage.go

```diff
@@ -35,9 +35,9 @@ type EpochCommitmentStorage struct {
 	shutdownOnce sync.Once
 }
 
-type epochDiffStorage struct {
-	spent   *objectstorage.ObjectStorage[*ledger.OutputWithMetadata]
-	created *objectstorage.ObjectStorage[*ledger.OutputWithMetadata]
+type EpochDiffStorage struct {
+	Spent   *objectstorage.ObjectStorage[*ledger.OutputWithMetadata]
+	Created *objectstorage.ObjectStorage[*ledger.OutputWithMetadata]
```

Comment:
Do we use these outside the package?

Reply:
Not anymore, forgot to change it back after discarded idea